packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageContentStyles.ts

```diff
@@ -0,0 +1,56 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { defaultChatDensity } from '../../../../components/Chat/chatDensity';
+
+import { ChatMessageContentStylesProps } from '../../../../components/Chat/ChatMessageContent';
+import { ChatMessageVariables } from './chatMessageVariables';
+import { pxToRem } from '../../../../utils';
+
+const getChatMessageVariantStyles = (props: ChatMessageContentStylesProps) => {
+  const density = props.density || defaultChatDensity;
+  switch (true) {
+    case props.layout === 'refresh' && density === 'comfy':
+      return ({ props: p, variables: v, theme }): ICSSInJSStyle => ({
+        color: v.contentColor,
+        wordBreak: 'break-word',
+        wordWrap: 'break-word',
+        '& a': {
+          color: 'inherit',
+          textDecoration: 'underline',
+          wordBreak: 'break-all',
+          '&:hover': { textDecorationStyle: 'double' },
+          '&:focus': { textDecorationStyle: 'double' },
+        },
+        ...(p.failed && {
+          color: theme.siteVariables.colorScheme.default.foreground,
+        }),
+      });
+    case density === 'comfy':
+      return ({ props: p }): ICSSInJSStyle => ({
+        ...(p.hasBadge && p.badgePosition === 'end' && { marginRight: pxToRem(4) }),
+      });
+
+    default:
+      return () => ({});
+  }
+};
+
+export const chatMessageContentStyles: ComponentSlotStylesPrepared<
+  ChatMessageContentStylesProps,
+  ChatMessageVariables
+> = {
+  root: (componentStyleFunctionParam): ICSSInJSStyle => {
+    const { props: p, variables: v } = componentStyleFunctionParam;
+    return {
+      color: v.contentColor,
+      display: 'block',
+      '& a': {
+        outline: 'none',
+        color: p.mine ? v.linkColorMine : v.linkColor,
+        ':focus': {
+          textDecoration: 'underline',
+        },
+      },
+      ...getChatMessageVariantStyles(p)(componentStyleFunctionParam),
+    };
+  },
```

Comment:
```suggestion
  root: ({ props: p, variables: v }): ICSSInJSStyle => {
    return {
      color: v.contentColor,
      display: 'block',
      '& a': {
        outline: 'none',
        color: p.mine ? v.linkColorMine : v.linkColor,
        ':focus': {
          textDecoration: 'underline',
        },
      },
      ...getChatMessageVariantStyles(p)(componentStyleFunctionParam),
    };
  },
```

nit

Reply:
Oh I needed `componentStyleFunctionParam` on L66 for `getChatMessageVariantStyles` so I didn't use the spread

Reply:
Ah nevermind I saw the next comment so removing closure and keep spread :)